### PR TITLE
Rib 81 implement login and registration page

### PR DESCRIPTION
**Pull request by VladislavChval**

### CODE REVIEW

**Review by dondowoldo — CHANGES_REQUESTED** [review]
Things that need to be changed : 
- Maybe merge develop branch into this branch so fragments can be used on the header
- Remove "type=text" as per comment
- Looks fine on full resolution PC, but on mobile header is squished and cards not centered
- Maybe slightly larger cards. They take about 50% of mobile screen now and the fields seem really really small. At least on my device.
- Create separate JS file for the script and just link it in the html

**Review by MatejFrnka — APPROVED** [review]

**Review by MatejFrnka — APPROVED** [review]
fix the duplicated type, rebase to the latest updates by doing:

To be able to use fragments, do this:
git checkout develop
git pull
git checkout %this branch%
git rebase develop

**Comment by vdrfg** [comment]
- Reworked design
- Icons still missing
- Login doesn't work for some reason

File: src/main/resources/templates/user/loginForm.html [hunk 1]
```diff
@@ -1,61 +1,53 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" type="text/css" th:href="@{/styles/index.css}">
+    <link href="https://fonts.googleapis.com/css2?family=Andika:wght@400;700&family=Montserrat:wght@200;400;600&family=Righteous&family=Ubuntu:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+
+</head>
```
Comment (marthubner) [inline src/main/resources/templates/user/loginForm.html:12]: The head is missing favicon and the title should probably be "Login | Ratespot" to be consistent with the other pages.
Maybe try to use our predefined fragment.

File: src/main/resources/templates/user/signupForm.html [hunk 2]
```diff
@@ -1,19 +1,62 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" type="text/css" th:href="@{/styles/index.css}">
+    <link href="https://fonts.googleapis.com/css2?family=Andika:wght@400;700&family=Montserrat:wght@200;400;600&family=Righteous&family=Ubuntu:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+</head>
```
Comment (marthubner) [inline src/main/resources/templates/user/signupForm.html:11]: The head is missing favicon and also the title should not be login.
Maybe try to use our predefined fragment.

**Review by marthubner — CHANGES_REQUESTED** [review]

**Review by marthubner — APPROVED** [review]

**Review by dondowoldo — APPROVED** [review]

**Pull request merged**